core/pdf.php

```diff
@@ -323,7 +323,7 @@ function pdfexport($WHERE)
 	$pdf->SetRightMargin($right_margin);
 
 	// add downscaling
-	if ($config['pdf_scale'])
+	if (array_key_exists('pdf_scale', $config))
```

Comment:
`if ($config['pdf_scale'])` will only evaluate to true if it is defined and has a truthy value.
Your edit will change that to only if it is defined. Please change to something like:
`if (array_key_exists('pdf_scale', $config) && $config['pdf_scale'])` to keep current functionality while removing the warning.

Reply:
code amemded